bootstrap/service.go

```diff
@@ -176,57 +181,61 @@ func (bs bootstrapService) Add(ctx context.Context, token string, cfg Config) (C
 }
 
 func (bs bootstrapService) View(ctx context.Context, token, id string) (Config, error) {
-	owner, err := bs.identify(ctx, token)
+	user, err := bs.identify(ctx, token)
 	if err != nil {
 		return Config{}, errors.Wrap(svcerr.ErrAuthentication, err)
 	}
-	cfg, err := bs.configs.RetrieveByID(ctx, owner, id)
+
+	// Retrieve the bootstrap configuration.
```

Comment:
Same as things service we need to authorize the users has `ViewPermission` to the thing if the boostrap config has thingID

Reply:
This also applies to update operations `EditPermission`